test/binary/function-local-count-zero-dump.txt

```diff
@@ -0,0 +1,47 @@
+;;; TOOL: run-objdump-gen-wasm
+;;; ARGS: -x
```

Comment:
Maybe put this file under `test/dump/`

Reply:
ok, done, but this file is very similar `test/binary/function-local-count-zero.txt` that already exists in there, but it uses `TOOL: run-gen-wasm`.

Reply:
Hmm... I see.   Maybe just update that test to use `run-objdump-gen-wasm`?  Would be loosing any coverage by doing that?

Reply:
Yeah, I think so, it'd be great if they could both be done in one file but in lieu of that a separate file seemed sensible.

What would be lost is coverage of the ability to convert this:

```
  func {
    local_decls[4]
    locals[1] i64
    locals[0] i32
    locals[2] f64
    locals[0] f32
  }
```

to this:

```
  (func (;1;) (type 0)
    (local i64 f64 f64)))
```

i.e. ignoring the zero-locals when generating the text format.

So it's a related problem, but more strict bounds since it would be invalid to have them in the text format even if they exist in the binary.

Reply:
It looks like you can add `;;; TOOL: run-objdump-gen-wasm` to the existing test and perhaps get the result you want without creating a new test file (tests seem to allow multiple `TOOL` lines)

Reply:
Oh, good call, I didn't think this was possible. Added now and it works.